pom.xml

```diff
@@ -703,7 +703,7 @@
       <id>scala-2.11</id>
       <activation>
         <property>
-          <name>!scala-2.12</name>
+          <name>scala-2.11</name>
         </property>
```

Comment:
If we don't specify a profile on the command line, does the build pickup a default one and build it properly?

Reply:
on travis is mandarory because of change-scala-version.sh

on local machine it works as scala version is defined on properties.. (version 2.11)